lib/rubocop/cop/layout/closing_heredoc_indentation.rb

```diff
@@ -49,18 +49,18 @@ module Layout
       class ClosingHeredocIndentation < Cop
         include Heredoc
 
+        SIMPLE_HEREDOC = '<<'.freeze
         MSG = '`%<closing>s` is not aligned with `%<opening>s`.'.freeze
         MSG_ARG = '`%<closing>s` is not aligned with `%<opening>s` or ' \
                   'beginning of method definition.'.freeze
 
         def on_heredoc(node)
+          return if heredoc_type(node) == SIMPLE_HEREDOC
```

Comment:
I find this a bit odd. Probably we should be returning some symbol for this method instead of the delimiter and probably we should also have this as a node extension instead. Not a big deal, but certainly an opportunity for improvement.

Reply:
@bbatsov I'm not exactly sure what you mean but if you provide more detail I can try to do it?

Reply:
See the `str_node.rb` file, where the `heredoc?` method is defined. Seems to me the type should also go there, although as I said it's not a big thing. I was thinking of some method `heredoc_type` that returns the type as a symbol.